tests/runtests.py

```diff
@@ -508,6 +510,11 @@ def paired_tests(paired_test, options, test_labels, parallel, start_at, start_af
         '-b', '--buffer', action='store_true',
         help='Discard output of passing tests.',
     )
+    parser.add_argument(
+        '--timing', action='store_true',
+        help='Output timings such as DB creation, DB cloning, DB teardown and'
+        ' total test run time.',
```

Comment:
Prefer "database" to "DB"?

```suggestion
        help='Output timings such as database creation, cloning, and teardown;'
        ' and total test run time.',
```